src/main/java/io/redlink/more/data/service/ElasticService.java

```diff
@@ -84,6 +89,41 @@ public List<String> storeDataPoints(final List<DataPoint> dataBulk, final Routin
         }
     }
 
+
+    public long deleteDataPoints(RoutingInfo routingInfo, String key, Set<String> filteredByValues) throws IOException {
+        if (filteredByValues.isEmpty() || key == null) {
+            return 0L;
+        }
+        final String indexName = "study_" + routingInfo.studyId();
+        DeleteByQueryRequest request = new DeleteByQueryRequest.Builder()
+                .index(indexName)
+                .query(q -> q
+                        .bool(b -> b
+                                .must(m -> m.term(t -> t.field("study_id")
```

Comment:
Diese Konstanten sollten auch in der gleichen Klasse als oben die GARMIN_SUMMARY_ID_KEY sein